From 3d94eb90dcd99fdb618a1cdc26a83c70c690a37c Mon Sep 17 00:00:00 2001
From: Dinesh Kumar Mirche <dmirche@habana.ai>
Date: Tue, 11 Apr 2023 16:08:09 +0300
Subject: [PATCH] [SW-103347] gaudi3: add boot info to arcevent

This patch adds the support of boot info module to arcevent
as this module is required for handling serr and derr interrupts.

Change-Id: I9f50056dfbca26c326930b53a43181ec4dca0222
Signed-off-by: Dinesh Kumar Mirche <dmirche@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index e569264..db8d8c4 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -117,6 +117,8 @@
 ZEPHYR_CONFIGURE_CMDS += && $(ZEPHYR_CONF_ENV) $(HOST_DIR)/bin/cmake $(ZEPHYR_CONF_OPTS) $(ZEPHYR_CONF_OPTS_EVENT) -B $(ZEPHYR_EVENT_DIR)
 ZEPHYR_BUILD_CMDS += && $(TARGET_MAKE_ENV) $(MAKE) -C $(@D)/$(ZEPHYR_EVENT_DIR)
 ZEPHYR_INSTALL_TARGET_CMDS += && $(INSTALL) -D -m 0644 $(@D)/$(ZEPHYR_EVENT_DIR)/zephyr/zephyr.elf $(BINARIES_DIR)/zephyr_event.elf
+ZEPHYR_EVENT_MODULES = boot_info
+$(foreach module,$(ZEPHYR_EVENT_MODULES),$(call DEFINE_MODULE_ENVVARS,$(module),$(ZEPHYR_EVENT_DIR),$(ZEPHYR_EVENT_NAME)))
 endif
 
 ifdef BR2_PACKAGE_ZEPHYR_ARC_PREBOOT
